include/common.sh

```diff
@@ -1330,6 +1330,24 @@ isURLReachable() {
   fi
 }
 
+checkAvailableSize()
+{ 
+  printInfo "Checking available size to install package."
+  
+  packageSize="$1"
+  partitionSize=$(df -m . | tail -1 | awk '{print $3}' | cut -f1 -d '/')
```

Comment:
```suggestion
  partitionSize=$(/bin/df -m . | tail -1 | awk '{print $3}' | cut -f1 -d '/')
```
There's also a coreutils df, so to be safe, let's use /bin/df